test/bdd/fixtures/scripts/vcs_configure.sh

```diff
@@ -115,7 +115,7 @@ until [ $n -ge $maxAttempts ]
 do
    responseCreatedTime=$(curl --header "Content-Type: application/json" \
    --request POST \
-   --data '{"name":"vc-issuer-didkey", "uri":"http://vc-issuer-didkey.com", "signatureType":"Ed25519Signature2018", "did":"did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd","didPrivateKey":"28xXA4NyCQinSJpaZdSuNBM4kR2GqYb8NPqAtZoGCpcRYWBcDXtzVAzpZ9BAfgV334R2FC383fiHaWWWAacRaYGs","signatureRepresentation":1}' \
+   --data '{"name":"vc-issuer-didkey", "uri":"http://vc-issuer-didkey.com", "signatureType":"Ed25519Signature2018", "did":"did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd","didPrivateKey":"28xXA4NyCQinSJpaZdSuNBM4kR2GqYb8NPqAtZoGCpcRYWBcDXtzVAzpZ9BAfgV334R2FC383fiHaWWWAacRaYGs","signatureRepresentation":1,"didKeyType":"Ed25519"}' \
```

Comment:
We don't need didKeyType when DID is passed. DIDKey is used only when edge-service needs to create a trustBloc DID.

Reply:
ok

Reply:
done